osu.Game/Skinning/Editor/SkinEditor.cs

```diff
@@ -342,7 +344,7 @@ public void Save()
                 currentSkin.Value.UpdateDrawableTarget(t);
 
             skins.Save(skins.CurrentSkin.Value);
-            onScreenDisplay?.Display(new SkinEditorToast(ToastStrings.SkinSaved, currentSkin.Value.SkinInfo.ToString()));
+            onScreenDisplay?.Display(new SkinEditorToast(ToastStrings.SkinSaved, currentSkin.Value.SkinInfo.ToString() ?? "Unknown"));
```

Comment:
Not sure I would have bothered to fall back with this one. It's only an issue because this is `object.ToString()` which is allowed to return `string?`. In practice a `Live<SkinInfo>` should never return null from `ToString()`.

Reply:
What would you prefer in this scenario? `.AsNonNull()`? Seems like one which is going to come up quite regularly..

Reply:
Probably `.AsNonNull()` yeah. Not that it matters much.